packages/next/build/analysis/get-page-static-info.ts

```diff
@@ -184,6 +185,10 @@ function getMiddlewareConfig(
     result.matchers = getMiddlewareMatchers(config.matcher, nextConfig)
   }
 
+  if (typeof config.regions === 'string' || Array.isArray(config.regions)) {
+    result.regions = config.regions
+  }
```

Comment:
**suggestion:** we could warn them (or even fail) when `config.regions` is defined but with the wrong values.

I'll vote for warning, as we do [here](https://github.com/vercel/next.js/pull/40881/files#diff-cb7ac6392c3dd707c5edab159c3144ec114eafea92dad5d98f4eedfc612174d2R233)

Reply:
I think a warning would definitely be good so users aren't confused why the config isn't applying when incorrect

Reply:
Do we want to warn or throw?

Reply:
+1 on the Warn team